src/App.vue

```diff
@@ -168,605 +132,31 @@
-    const e = event || window.event;
-
-    if (e.keyCode === 27) { // Escape key
-      closeAllModals();
-    }
-  });
-});
-
-</script>
-
-<script>
-import { defineComponent, reactive, ref, computed } from "vue"
-import * as vNG from "v-network-graph"
-import {
-  ForceLayout,
-} from "v-network-graph/lib/force-layout"
-
-const original_nodes = {
-  node1: { name: "Node 1" },
-  node2: { name: "Node 2" },
-  node3: { name: "Node 3" },
-  node4: { name: "Node 4" },
-}
-var original_edges = {
-  edge1: { source: "node1", target: "node2" },
-  edge2: { source: "node2", target: "node3" },
-  edge3: { source: "node3", target: "node4" },
-}
-const graph = ref();
-const nodes = reactive({ ...original_nodes })
-const edges = reactive({ ...original_edges })
-const layouts = ref({
-  nodes: {
-    node1: { x: 0, y: 0 },
-    node2: { x: 50, y: 50 },
-    node3: { x: 100, y: 0 },
-    node4: { x: 150, y: 50 },
-  },
-});
-
-const nextNodeIndex = ref(Object.keys(nodes).length + 1)
-const nextEdgeIndex = ref(Object.keys(edges).length + 1)
-
-const selectedNodes = ref([]);
-const selectedEdges = ref([]);
-const eventLogs = reactive([])
-const defaultQuery = `MATCH (p)-[e]->(q) RETURN e LIMIT 100
-UNION ALL
-MATCH (p) RETURN p as e LIMIT 100`
-const queryRef = ref(defaultQuery)
-
-const dataPanel = ref({})
-const commandPanel = ref([])
-async function writeTransaction(query, params, callback) {
-  console.log(query, params)
-  const session = document.driver.session({
-    database: 'neo4j',
-  })
-  // Create a node within a write transaction
-  const res = await session.writeTransaction(tx => {
-    return tx.run(query, params)
-  })
-  // Close the sesssion
-  session.close()
-  if (callback) callback(res);
-  return res
-}
-function updateDataPanel(objInfo, obj) { // TODO actually can just change the params to obj
-  const total = selectedNodes.value.length + selectedEdges.value.length
-  if (document.fixDataPanel && total > 0) {
-    return
-  }
-  if (!objInfo) return;
-  dataPanel.value = {}
-  if (Object.prototype.hasOwnProperty.call(objInfo.__proto__, '__isRelationship__')) {
-    dataPanel.value['objType'] = 'Relationship'
-    dataPanel.value['labels'] = [objInfo.type];
-  } else if (Object.prototype.hasOwnProperty.call(objInfo.__proto__, '__isNode__')) {
-    dataPanel.value['objType'] = 'Node'
-    dataPanel.value['labels'] = objInfo.labels;
-  }
-  if (obj) {
-    dataPanel.value['obj'] = obj
-  }
-  dataPanel.value['id'] = objInfo.identity.toNumber();
-  dataPanel.value['properties'] = objInfo.properties;
-}
-
-
+import graphData from './graphData'
+
+const dataPanel = {};
+const graph = graphData.graph
+const nodes = graphData.nodes
+const edges = graphData.edges
+const configs = graphData.configs
+const layouts = graphData.layouts
+const selectedNodes = graphData.selectedNodes
+const selectedEdges = graphData.selectedEdges
+const d3ForceEnabled = graphData.d3ForceEnabled
+const eventHandlers = graphData.eventHandlers
```

Comment:
Is this a requirement by `v-network-graph`? Otherwise, it would be cleaner to shove them into a dictionary